logging/bin/deploy_osd_content.sh

```diff
@@ -45,7 +45,9 @@ log_info "Configuring OpenSearch Dashboards...this may take a few minutes"
 
 # wait for pod to show as "running" and "ready"
 log_info "Waiting for OpenSearch Dashboards pods to be ready ($(date) - timeout 10m)"
-kubectl -n $LOG_NS wait pods --selector "app=opensearch-dashboards"  --for condition=Ready --timeout=10m
+osdlabels="$(kubectl -n $LOG_NS get deployment v4m-osd -o=jsonpath='{.spec.selector.matchLabels}'|sed s/{//g|sed s/}//g|sed s/\"//g|sed s/:/=/g)"
```

Comment:
Those `sed` calls are kind of hard to read.  We could change them to make the line:
 `osdlabels="$(kubectl -n $LOG_NS get deployment v4m-osd -o=jsonpath='{.spec.selector.matchLabels}'| tr -d '{}"' | tr : '=')`. Is it worth it?